src/pages/_app.tsx

```diff
@@ -34,17 +36,20 @@ type Props = AppProps & ReduxWrapperAppProps<State>;
 const CustomApp = (appProps: Props): JSX.Element => {
   const { Component, pageProps, store } = appProps;
   return (
-    <Provider store={store}>
-      <ContextWrapper>
-        <AuthProvider>
-          <UserProfileProvider>
-            <Core>
-              <Component {...pageProps} />
-            </Core>
-          </UserProfileProvider>
-        </AuthProvider>
-      </ContextWrapper>
-    </Provider>
+    <>
+      <GlobalStyle />
```

Comment:
Adding these styled requires additions in `_document.tsx` to have the CSS for initial page load with SSR

Example of it can be found here: https://github.com/dotkom/gallery/blob/master/src/pages/_document.tsx#L5-L29